flake.nix

```diff
@@ -1,65 +1,48 @@
 
   inputs = rec {
-    nixpkgs-unstable = { url = "github:nixos/nixpkgs/release-22.11"; };
+    nixpkgs = { url = "github:nixos/nixpkgs/release-22.11"; };
     # nixpkgs-github-runner = { url = "github:nixos/nixpkgs/941c79b6207fa84612b4170ca3bc04984f3d79fc"; };
-    nixpkgs-github-runner = nixpkgs-unstable;
-
-    steveejKeys = { url = "https://github.com/steveej.keys"; flake = false; };
-    jost-sKeys = { url = "https://github.com/jost-s.keys"; flake = false; };
-    maackleKeys = { url = "https://github.com/maackle.keys"; flake = false; };
-    neonphogKeys = { url = "https://github.com/neonphog.keys"; flake = false; };
-    thedavidmeisterKeys = { url = "https://github.com/thedavidmeister.keys"; flake = false; };
-    zippyKeys = { url = "https://github.com/zippy.keys"; flake = false; };
+    nixpkgs-github-runner = nixpkgs;
+
+    # nix darwin
+    darwin.url = "github:lnl7/nix-darwin/master";
+    darwin.inputs.nixpkgs.follows = "nixpkgs";
+
+    # home manager
+    home-manager.url = "github:nix-community/home-manager";
+    home-manager.inputs.nixpkgs.follows = "nixpkgs";
+
+    keys_steveej = { url = "https://github.com/steveej.keys"; flake = false; };
+    keys_jost-s = { url = "https://github.com/jost-s.keys"; flake = false; };
+    keys_maackle = { url = "https://github.com/maackle.keys"; flake = false; };
+    keys_neonphog = { url = "https://github.com/neonphog.keys"; flake = false; };
+    keys_thedavidmeister = { url = "https://github.com/thedavidmeister.keys"; flake = false; };
+    keys_zippy = { url = "https://github.com/zippy.keys"; flake = false; };
+    keys_davhau = { url = "https://github.com/davhau.keys"; flake = false; };
   };
 
-  outputs = inputs: {
-    magicPaths = {
-      githubRunnerHraTokenHostPath = "/var/secrets/github-runner/hra2.token";
-      githubRunnerHraTokenMountPoint = "/secrets/github-runner/token";
-    };
-
-    nixosConfigurations = rec {
-      githubRunnerContainerPathFn = name: extraLabels: (inputs.nixpkgs-github-runner.lib.nixosSystem {
-        system = "x86_64-linux";
-        modules = [
-          ./nixos-containers/github-runner/configuration.nix
-        ];
+  outputs = inputs@{ flake-parts, ... }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
 
-        specialArgs = {
-          githubRunnerHolochainHolochainTokenFile = inputs.self.magicPaths.githubRunnerHraTokenMountPoint;
-          inherit name extraLabels;
-        };
-      }).config.system.build.toplevel;
+      # auto import all nix code from `./modules`
+      imports = map (m: "${./modules}/${m}")
+        (builtins.attrNames (builtins.readDir ./modules));
 
-      githubRunnerHost = inputs.nixpkgs-unstable.lib.nixosSystem {
-        system = "x86_64-linux";
-        modules = [
-          ./nixos-hosts/github-runner-host/configuration.nix
-        ];
+      systems = [ "x86_64-linux" ];
```

Comment:
We can add other systems here, right?

Reply:
Sure we can. Done in https://github.com/holochain/holochain-infra/pull/3/commits/504f22ec89657045284f34ba2491daacaf41fbc3